metric/activeusers/classes/metric.php

```diff
@@ -18,10 +18,12 @@
 
 use tool_cloudmetrics\metric_item;
 
+use block_online_users\fetcher;
```

Comment:
copy pasta?

Reply:
ah, I originally thought this was a typo but it's not. The reason I was confused is a language subtly, there is the concept of 'active users' which is the long term trend of how many users are active on the site in the last 30 days which you can pull from here:

https://github.com/moodle/moodle/blob/master/lib/classes/hub/registration.php#L157

Then there is the metric for 'online users' which is what this patch. So this patch is totally correct, and also a good priority, but it's actually issue #3 not issue #2. 

So keep working on this but rename it and realign it (might be easier as new pr)